src/encoded/commands/create_annotated_filenames.py

```diff
@@ -816,7 +824,9 @@ def get_analysis(
         software_and_versions, reference_genome_code
     )
     if file_format_utils.is_chain_file(file_extension):
-        value = f"{value}{ANALYSIS_INFO_SEPARATOR}{get_chain_file_value(file)}"
+        value = ANALYSIS_INFO_SEPARATOR.join([value,get_chain_file_value(file)]) if value else get_chain_file_value(file)
+    if file_format_utils.is_fasta_file(file_extension):
```

Comment:
Not quite sure I understand. You are only testing for fasta file here. Do you always want to append`dsa` in that case?

Reply:
Great point. I'll add a specification for only DSA fasta files